src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/help.jelly

```diff
@@ -54,6 +54,78 @@ THE SOFTWARE.
     </p>
     <pre>stage name: 'Build', concurrency: 1</pre>
     <p>
-        There are also some predefined variables, shown at the bottom.
+        Many steps require complex nested configuration for a parameter.
+        (And some nested configuration objects in turn have object-typed parameters.)
+        There are three ways these nested objects may be specified:
     </p>
+    <ol>
+        <li>
+            <p>
+                As maps of parameters. Default values may be omitted.
+                (Note that Groovy map literal syntax resembles list syntax in using square brackets, but keys and values are separated by colons.)
+            </p>
+            <p>
+                The special map key <code>$class</code> is used to represent the simple or (where necessary) fully-qualified class name
```

Comment:
I think we should explain the better syntax first and position this as a fallback.